ERROR at //build/config/BUILDCONFIG.gn:700:11: Unable to load "/chromium-141.0.7390.37/buildtools/third_party/libc++/BUILD.gn".
          "//buildtools/third_party/libc++:libcxx_headers",
          ^-----------------------------------------------

--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -617,7 +617,7 @@ foreach(_target_type,
 
       if (use_libcxx_modules) {
         # Add a dependency needed for Clang modules builds.
-        deps += [ "//buildtools/third_party/libc++:all_modules" ]
+        deps += [ ]
 
         if (!defined(invoker.generate_modulemap) &&
             defined(generate_modulemap)) {
@@ -748,7 +748,7 @@ foreach(_target_type,
 
       if (use_libcxx_modules) {
         # Add a dependency needed for Clang modules builds.
-        deps += [ "//buildtools/third_party/libc++:all_modules" ]
+        deps += [ ]
 
         if (!defined(invoker.generate_modulemap) &&
             defined(generate_modulemap)) {
--- a/build/modules/BUILD.gn
+++ b/build/modules/BUILD.gn
@@ -5,8 +5,6 @@
 import("//build/config/c++/c++.gni")
 import("//build/config/clang/clang.gni")
 import("//build/config/sysroot.gni")
-import("//buildtools/third_party/libc++/libcxx_headers.gni")
-import("//buildtools/third_party/libc++/modules.gni")
 if (is_win) {
   import("//build/toolchain/win/win_toolchain_data.gni")
 }
@@ -53,9 +51,7 @@ if (use_unified_system_module) {
     }
   }
 
-  system_headers = libcxx_headers + [
-                     "//buildtools/third_party/libc++/__config_site",
-                     "//buildtools/third_party/libc++/__assertion_handler",
+  system_headers = [
                    ]
   root_include_dir = ""
   extra_headers = []
@@ -77,7 +73,6 @@ if (use_unified_system_module) {
   }
 
   if (root_include_dir != "") {
-    system_headers += expand_directory("${sysroot}/${root_include_dir}", true)
   }
 
   modulemap = "${target_gen_dir}/module.modulemap"
--- a/v8/BUILD.gn
+++ b/v8/BUILD.gn
@@ -7417,7 +7417,6 @@ v8_component("v8_libbase") {
     deps += [ ":libm" ]
   }
 
-  deps += [ "//third_party/llvm-libc:headers" ]
 
   # TODO(infra): Add support for qnx, freebsd, openbsd, netbsd, and solaris.
 }
